modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/UserCardTag.java

```diff
@@ -87,8 +87,8 @@ private void _populateContext() {
 			setSubtitle(_userCard.getSubtitle());
 		}
 
-		if (context.get("userCssColorClass") == null) {
-			setUserColorCssClass(_userCard.getUserColorCssClass());
+		if (context.get("userColorClass") == null) {
```

Comment:
hey @mateomustapic I gave it a look. I have a suggestion we can do, I think we can keep as we had done before, keep `userColorCssClass` for taglib. It seems that the problem was just here, that we were getting `userCssColorClass` instead of `userColorClass` like now. What do you think?

Reply:
I think that it won't work if we keep `userColorCssClass`, because the taglib setters must be the same as `userColorClass` (setUserColorClass). 

That is the main issue why this doesn't work, because if we change to `userColorCssClass`, the we must change this property here too. 
https://github.com/liferay/clay/blob/0ce6bf8fad67a642ae61170085c7bb83158a1669/packages/clay-card/src/ClayUserCard.soy#L29

Reply:
The only way I managed to make it work was doing it this way :/

Reply:
Hey @mateomustapic sorry for the delay, I had a lot of meetings today 😁. But I think it still allows us to keep the methods with `userColorCssClass` and only the context with `userColorClass` that is passed to the component. I will send this change to you to confirm that it will work.

Reply:
That would be the best, if it works, but when I change the context and the tld tag I get this error `Cannot find a setter method for the attribute userColorClass of the tag handler com.liferay.frontend.taglib.clay.servlet.taglib.soy.UserCardTag at org.apache.jasper.compiler.DefaultErrorHandler.jspError` because the method name should match the tld tag attribute.

Reply:
Hmm okay, no. I think I'll send Brian this and explain to him that we can't cause a breaking change now. @jbalsas can you help us decide here? just to make sure we're doing the right thing.

Reply:
Hey @matuzalemsteles, send this to me and I'll figure out what we need to do. It should be fine to deviate at the last moment when we move into Clay v2 territory.

Please, send the PR my way so I can look into it carefully!